src/sbt-test/sbt-avro/avscparser/build.sbt

```diff
@@ -0,0 +1,27 @@
+name := "avscparser-test"
+
+libraryDependencies ++= Seq(
+  "org.apache.avro" % "avro" % "1.10.0",
+  "org.specs2" %% "specs2-core" % "4.9.4" % Test
+)
+
+(Compile / avroSchemaParser) := {
+  class AnnotateWithArtifactSchemaParser(moduleID: ModuleID)
+      extends org.apache.avro.Schema.Parser {
+    override def parse(file: java.io.File): org.apache.avro.Schema = {
+      val schema = super.parse(file)
+      if (schema.getType == org.apache.avro.Schema.Type.RECORD) {
+        schema.addProp("com.cavorite.sbt-avro.artifact", moduleID.toString())
```

Comment:
What is the effect of this statement ?

Reply:
This is the test to demonstrate our use case - I would like to add custom property (avro allows un-recognized properties in schema, citation needed...) in this case the artifact where compiled schema is published, so data consumers can easily see how to include compiled dependency in their project. 
This particular case would generate schema like:
```json
{
  "type": "record",
  "name": "...",
  "namespace": "...",
  "fields": ["..."]
  "com.cavorite.sbt-avro.artifact": "avscparser-test:avscparser-test:0.1.0-SNAPSHOT"
}
```